.gitlab-ci.yml

```diff
@@ -82,3 +82,30 @@ staging:
     K8S_SECRET_ELASTIC_APM_SECRET_TOKEN: "$GL_QA_ELASTIC_APM_TOKEN"
     K8S_SECRET_ELASTIC_APM_SERVER_URL: "$GL_QA_ELASTIC_APM_URL"
     K8S_SECRET_ELASTIC_APM_SERVICE_NAME: "berths-backend"
+
```

Comment:
Could we remove the `test-codeanalysis:` part above?

Reply:
yes, if we don't like it...

Reply:
Let's remove it because it's already run by Travis, no need to run it again in this stage.